docs/cache_requirements.md

```diff
@@ -0,0 +1,79 @@
+# AMP Cache SXG Requirements
+
+## Audience
+
+The audience for this document is people intending on implementing their own AMP
+signed exchange generator, independent of amppackager, and those implementing
+their own AMP SXG cache for the purposes of privacy-preserving prefetch. Users
+of `amppkg` need not read this, as the tool should automatically guarantee the
+following requirements are met.
+
+## Google AMP cache
+
+The Google AMP cache sets some requirements in addition to the ones set by the
+[SXG spec](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html).
+These include:
+
+ * The signed `fallback URL` must equal the URL at which the SXG was delivered.
+ * The payload must be:
+   * non-empty.
+   * valid transformed AMP. The canonical definition of transformed AMP is the
+     return value of `transform.Process()`. If given a valid AMP doc as input,
```

Comment:
Should this link to the source? At least the file?

Reply:
Is there documentation on what "valid AMP" is? Does it make sense to link to the validator?

Reply:
Done and done. There is a [specification](https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml) that seems out of date, so I linked to the validator instead.